js/src/panes/pane.fullscreen.js

```diff
@@ -48,6 +48,7 @@ Wu.Fullscreen = Wu.Evented.extend({
 
 		// close trigger		
 		Wu.DomEvent.on(this._closer, 'click', this.destroy, this);
+		window.addEventListener('popstate', this.destroy.bind(this), false);
```

Comment:
Hum.. I think like this, `e` will be passed to `destroy()` - but destroy fn is currently getting `options` object as first arg. But it's not an issue from what I can see, but good to keep in mind.